libs/api/protocol/src/lib/mapper-profile/protocol-entity-document.mapper-profile.ts

```diff
@@ -0,0 +1,104 @@
+import {
+	Mapper,
+	MappingConfiguration,
+	createMap,
+	extend,
+	forMember,
+	mapFrom,
+	mapWith,
+} from '@automapper/core';
+import { AutomapperProfile, getMapperToken } from '@automapper/nestjs';
+import { Inject, Injectable } from '@nestjs/common';
+
+import { UserProducer } from '../core/entity/partials/producer-partial.entity';
+import {
+	CommunicationMessage,
+	CommunicationMessagePayload,
+} from '../core/entity/protocol-entries/communication-message.entity';
+import {
+	ProtocolCommunicationEntryBase,
+	ProtocolEntryBase,
+} from '../core/entity/protocol-entries/protocol-entry-base.entity';
+import {
+	CommunicationMessageDocument,
+	CommunicationMessagePayloadDocument,
+} from '../infra/schema/communication-message.schema';
+import { UserProducerDocument } from '../infra/schema/producer-partial.schema';
+import {
+	ProtocolCommunicationEntryBaseDocument,
+	ProtocolEntryBaseDocument,
+} from '../infra/schema/protocol-entry-base.schema';
+import { unitEntityToDocumentMapper } from './unit-partial.mapper-profile';
+
+export abstract class ProtocolBaseToDocument extends AutomapperProfile {
```

Comment:
This is separation by direction again and not separation by target class as you used in the deployment feature. I think we should decide on one paradigm for consistency.